src/integration/kotlin/pl/edu/agh/gem/integration/controller/ExpenseControllerIT.kt

```diff
@@ -536,5 +539,67 @@ class ExpenseControllerIT(
                 errors.first().code shouldBe UserNotParticipantException::class.simpleName
             }
         }
+
+        should("delete expense") {
+            // given
+            val expense = createExpense(id = EXPENSE_ID, groupId = GROUP_ID, creatorId = USER_ID)
+            repository.create(expense)
+            stubGroupManagerMembers(createGroupMembers(USER_ID, OTHER_USER_ID), GROUP_ID)
+
+            // when
+            val response = service.delete(createGemUser(USER_ID, EMAIL), GROUP_ID, EXPENSE_ID)
+
+            // then
+            response shouldHaveHttpStatus OK
+            repository.findByExpenseIdAndGroupId(EXPENSE_ID, GROUP_ID).also {
+                it.shouldBeNull()
+            }
+        }
+        should("return forbidden if user is not a group member") {
```

Comment:
why we check if use user is group meber when only creator can delete that ?

Reply:
as discussed on daily. This way we will not share the information if expense with given id exists or not